src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -109,24 +109,6 @@ static ExperimentalFeature()
                 new ExperimentalFeature(
```

Comment:
For those that are kept experimental, their tests need to be declared in `test\tools\TestMetadata.json`, otherwise their tests will not be executed.
Also, for `"PSForEachObjectParallel"` and `"PSPipelineChainOperators"`, they need to be removed from that file as they become official.